imod/tests/test_mf6/test_multimodel/test_mf6_partitioning_unstructured.py

```diff
@@ -338,20 +338,32 @@ def test_partitioning_unstructured_with_well(
 @parametrize_with_cases("partition_array", cases=PartitionArrayCases)
 def test_partition_transport(    tmp_path: Path,
     circle_model_transport: Modflow6Simulation,partition_array: xu.UgridDataArray):
-
     circle_model_transport.write(tmp_path)
     circle_model_transport.run()
     concentration = circle_model_transport.open_concentration()
+    budget = circle_model_transport.open_transport_budget(["salinity"])
 
     new_circle_model = circle_model_transport.split(partition_array)
```

Comment:
In terms of naming, I think clearer for tests is to use a ``"expected"`` and ``"actual"`` prefix instead of ``""`` and ``"new"`` prefix. This reads clearer:

budget -> expected_budget
new_budget -> actual_budget

Alternatively, you can use ``"unsplit"`` and ``"split"`` or something. Or "full" and "split" as we do in some other tests (still not really happy with the prefix "full", so I'm open to better ideas!)

Reply:
done (actual and expected)